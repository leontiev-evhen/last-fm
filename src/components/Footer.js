import React from 'react';

export function Footer() {
	return (
		<footer>
			<div className="footer-section section-title">
				<h2>Last fm</h2>
			</div>
		</footer>
	);
}
