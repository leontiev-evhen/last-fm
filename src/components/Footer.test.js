import React from 'react';
import { mount } from 'enzyme';
import { Footer } from './Footer';

describe('Footer component', () => {
	it('render correctly Footer component', () => {
		const component = mount(<Footer />);
		expect(component).toMatchSnapshot();
	});
});
