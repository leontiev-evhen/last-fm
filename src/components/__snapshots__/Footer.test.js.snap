// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render correctly Footer component 1`] = `
ReactWrapper {
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <footer>
        <div
          class="footer-section section-title"
        >
          <h2>
            Last fm
          </h2>
        </div>
      </footer>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer-section section-title"
        >
          <h2>
            Last fm
          </h2>
        </div>,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": <div
            class="footer-section section-title"
          >
            <h2>
              Last fm
            </h2>
          </div>,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h2>
              Last fm
            </h2>,
            "className": "footer-section section-title",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": <h2>
                Last fm
              </h2>,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Last fm",
              },
              "ref": null,
              "rendered": Array [
                "Last fm",
              ],
              "type": "h2",
            },
          ],
          "type": "div",
        },
      ],
      "type": "footer",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <footer>
          <div
            class="footer-section section-title"
          >
            <h2>
              Last fm
            </h2>
          </div>
        </footer>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="footer-section section-title"
          >
            <h2>
              Last fm
            </h2>
          </div>,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": <div
              class="footer-section section-title"
            >
              <h2>
                Last fm
              </h2>
            </div>,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h2>
                Last fm
              </h2>,
              "className": "footer-section section-title",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": <h2>
                  Last fm
                </h2>,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Last fm",
                },
                "ref": null,
                "rendered": Array [
                  "Last fm",
                ],
                "type": "h2",
              },
            ],
            "type": "div",
          },
        ],
        "type": "footer",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
